### PR TITLE
Change laguage keywords and icon list to X Macro list generation, add suppport for M115 gcode for auto-setup & other bug fixes

### PR DESCRIPTION
**Pull request by guruathwal**
- Change Language Keywords & Icon list to auto-generate X-Macros Lists for easier keywords management #359 suggestion by @Andrei-Pozolotin
- Renamed icons to match icon list names.
- Add support for M115 G-code for auto-setup of TFT Controller features like AutoLevel and M27 auto-report
- Add updating of actual Feed rate and Flow rate from the printer.
- Bug Fix for bed temperature not displaying on receiving M115 reply.
- Improve List-view custom value display.
- Bug Fix Infobox display on the boot screen.
- Change chars icons to UTF character codes.
- Rename icons to match icon names in firmware.
- Other minor typo fixes.
- Increase version number to 25.3

### CODE REVIEW

File: TFT/src/User/API/icon_list.inc [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/**
+* Language Kewords Generation
```
Comment (Andrei-Pozolotin) [inline TFT/src/User/API/icon_list.inc]: you mean "icon" not "language"

**Comment by Andrei-Pozolotin** [comment]
@guruathwal 
1. looks good
2. do you plan `ICONCHAR` migration to X-Macro?

File: TFT/src/User/API/icon_list.inc (hunk 1 above)
Comment (guruathwal) [reply TFT/src/User/API/icon_list.inc]: Oops... Yes it should be icon
```suggestion
* Icon Keywords Generation
```

**Comment by guruathwal** [comment]
> @guruathwal
> 
> 1. looks good
> 2. do you plan `ICONCHAR` migration to X-Macro?

@Andrei-Pozolotin yes. Tomorrow.

**Comment by Andrei-Pozolotin** [comment]
I am curious, regarding: https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/pull/456/commits/841acdd84b3e7543e56bc16070bd2f8659b82c0f

would it not be better to keep ICONCHAR (name, image, color) as one record, i.e.:
* [TFT/src/User/API/UI/symbol.inc](https://github.com/random-builder/bigtree-tft-code/blob/custom-marlin/TFT/src/User/API/UI/symbol.inc)
* [TFT/src/User/API/UI/symbol.h](https://github.com/random-builder/bigtree-tft-code/blob/custom-marlin/TFT/src/User/API/UI/symbol.h)
* [TFT/src/User/API/UI/symbol.c](https://github.com/random-builder/bigtree-tft-code/blob/custom-marlin/TFT/src/User/API/UI/symbol.c)

**Pull request merged**